src/controllers/trust.historical.beneficial.owner.controller.ts

```diff
@@ -1,144 +1,10 @@
 import { NextFunction, Request, Response } from 'express';
-import * as config from '../config';
-import { logger } from '../utils/logger';
-import { TrusteeType } from '../model/trustee.type.model';
-import { getApplicationData, setExtraData } from '../utils/application.data';
-import { getTrustByIdFromApp, saveHistoricalBoInTrust, saveTrustInApp } from '../utils/trusts';
-import { mapBeneficialOwnerToSession, mapFormerTrusteeFromSessionToPage } from '../utils/trust/historical.beneficial.owner.mapper';
-import * as CommonTrustDataMapper from '../utils/trust/common.trust.data.mapper';
-import { ApplicationData } from '../model';
-import * as PageModel from '../model/trust.page.model';
-import { saveAndContinue } from '../utils/save.and.continue';
-import { Session } from '@companieshouse/node-session-handler';
-import { FormattedValidationErrors, formatValidationError } from '../middleware/validation.middleware';
-import { validationResult } from 'express-validator';
-import { safeRedirect } from '../utils/http.ext';
+import { getTrustFormerBo, postTrustFormerBo } from '../utils/trust.former.bo';
 
-const HISTORICAL_BO_TEXTS = {
-  title: 'Tell us about the former beneficial owner',
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  await getTrustFormerBo(req, res, next, false);
```

Comment:
does it need await for get here? Update controller doesn't have await.

Reply:
Ah you're right - I had it in both initially and forgot to remove it from here 👍

Reply:
Sorted 👍